backend/src/common/models/common/error.ts

```diff
@@ -0,0 +1,5 @@
+export interface IError {
+  msg: string
+  status: number
+  code?: number
+}
```

Comment:
Let's make error handling via class. You need to create a class CustomError that will accept in its constructor msg, status and code. Then we will be able to throw errors like this: `throw new CustomError(...)`.